stratum/hal/bin/barefoot/docker/Dockerfile.runtime

```diff
@@ -35,29 +35,14 @@ ENV KERNEL_HEADERS_PATH=/usr/src/kernel-headers
 ENV KDRV_DIR=/bf-sde/pkgsrc/bf-drivers/kdrv/bf_kdrv
 RUN /build-kdrv.sh
 
-# Without ONLP
-RUN bazel build //stratum/hal/bin/barefoot:stratum_bf \
-    --define sde_ver=$(cat $SDE_INSTALL/share/VERSION) \
-    --define phal_with_onlp=false && \
-    cp bazel-bin/stratum/hal/bin/barefoot/stratum_bf /tmp/stratum_bf_no_onlp
-
-# With ONLP
-RUN bazel build //stratum/hal/bin/barefoot:stratum_bf \
-    --define sde_ver=$(cat $SDE_INSTALL/share/VERSION)
+ENV OUTPUT_BASE /output
+ARG WITH_ONLP=true
 
-ENV STRATUM_BF_BASE stratum/hal/bin/barefoot
-RUN mkdir -p $OUTPUT_BASE/bin && \
-    cp bazel-bin/$STRATUM_BF_BASE/stratum_bf $OUTPUT_BASE/bin/ && \
-    cp /tmp/stratum_bf_no_onlp $OUTPUT_BASE/bin/ && \
-    cp $STRATUM_BF_BASE/platforms/*.json $OUTPUT_BASE/share/stratum/ && \
-    cp $STRATUM_BF_BASE/docker/stratum-entrypoint.sh /output/ && \
-    cp $STRATUM_BF_BASE/docker/stratum.flags $OUTPUT_BASE/share/stratum/ && \
-    cp $STRATUM_BF_BASE/*.conf $OUTPUT_BASE/share/stratum/ && \
-    touch $OUTPUT_BASE/share/stratum/phal_config.pb.txt && \
-    cp $STRATUM_BF_BASE/chassis_config.pb.txt $OUTPUT_BASE/share/stratum/ && \
-    mkdir -p /output/stratum/hal/lib/common/ && \
-    cp stratum/hal/lib/common/gnmi_caps.pb.txt /output/stratum/hal/lib/common/
-    # TODO(Yi Tseng): Remove two lines above when we are able to generate cap response automatically
+# Stratum package
+RUN bazel build //stratum/hal/bin/barefoot:stratum_bf_deb \
+    --define sde_ver=$(cat $SDE_INSTALL/share/VERSION) \
+    --define phal_with_onlp=$WITH_ONLP
+RUN dpkg -x bazel-bin/stratum/hal/bin/barefoot/stratum-bf_0.0.1_amd64.deb $OUTPUT_BASE
```

Comment:
I think there is also a deb file without the version. So we don't have to change this later.